data/4_to_be_checked/need-ms-parts/419.xml

```diff
@@ -369,6 +372,104 @@
               </bibl>
             </listBibl>
           </additional>
+          </msPart>
+          <msPart n="2" xml:id="Part2">
+            <msIdentifier>
+              <idno type="URI">http://syriaca.org/manuscript/1454</idno>
+              <altIdentifier>
+                <idno type="BL-Shelfmark">Add MS 14651 lower text</idno>
+              </altIdentifier>
+              <altIdentifier>
+                <collection>William Wright, Catalogue of the Syriac Manuscripts in the British Museum Acquired since the Year 1838</collection>
+                <idno type="Wright-BL-Arabic">948</idno>
+              </altIdentifier>
+              <altIdentifier>
+                <collection>William Wright, Catalogue of the Syriac Manuscripts in the British Museum Acquired since the Year 1838</collection>
+                <idno type="Wright-BL-Roman">DCCCCXLVIII</idno>
+              </altIdentifier>
+            </msIdentifier>
+            <msContents>
+              <summary/>
+              <textLang mainLang="syr">Syriac</textLang>
+              <msItem xml:id="p2a1" n="1" defective="true">
+                <locus from="1" to="217"/>
+                <title xml:lang="en">A manuscript of the Gospels, according to the Peshitta version</title>
+                <note>Most of these things, however, being written with red paint, have either nearly or altogether disappeared.</note>
+                <msItem xml:id="p2b1" n="2" defective="true">
+                  <title xml:lang="en">The Ammonian sections and Eusebian canons</title> 
+                  <note>These are marked on the margins, e. g. <locus from="5a">foll. 5 a</locus>, <locus from="48a">48 a</locus>, <locus from="143a">143 a</locus>, <locus from="182b">182 b</locus></note>
+                </msItem>
+                <msItem xml:id="p2b2" n="3" defective="true">
+                  <title xml:lang="en">A harmony at the foot of each page</title>
+                  <note>e. g. <locus from="48a">foll. 48 a</locus>, <locus from="88a">88 a</locus>, <locus from="92b">92 b</locus>,
+                    <locus from="151a">151 a</locus>.</note>
+                </msItem>
+                  <msItem xml:id="p2b3" n="4" defective="true">
+                    <title xml:lang="en">rubrics indicating lessons for various occasions</title>
+                    <note>e. g. <locus from="3a">foll. 3 a</locus>, <locus from="6b">6 b</locus>, <locus from="66a">66 a</locus>, <locus from="143a">143 a</locus>.</note> 
+                  </msItem>
+                <msItem xml:id="p2b4" n="5" defective="true">
+                  <locus from="112b">Fol. 112b</locus>
+                  <title>a Cross within an oval nimbus, which was probably once decorated with various
```

Comment:
I think this should go as an addition? @davidamichelson can you remind me how we are encoding decorations? I think we decided to deprecate using tei:decoNote elements in the tei:decoDesc? Or was it rare enough that we don't include it on the form but would want to use if needed?

This also applies to item p2b5, starting on l. 416